ui/app/templates/partials/status/cluster.hbs

```diff
@@ -75,9 +75,10 @@
         </div>
         <ul class="menu-list">
           <li class="action">
-            {{#link-to "vault.cluster.license" activeClusterName id="license" invokeAction=onLinkClick}}
+            {{#link-to "vault.cluster.license" activeCluster.name invokeAction=onLinkClick}}
```

Comment:
huh, I thought I had fixed this previously but I guess not! Should we also hide this link in OSS? Currently it's there, but will just redirect you immediately back to secrets.

Reply:
Yeah, I thought so too, but I've been meaning to fix the other small things anyway.

It does have a `{{#unless isOSS}}` around it, it's just hidden in the diff